rc-events-service/src/main/scala/com/epam/random_coffee/events/services/impl/EventServiceImpl.scala

```diff
@@ -1,37 +1,61 @@
 package com.epam.random_coffee.events.services.impl
 
-import com.epam.random_coffee.events.model.{ Event, EventId }
+import com.epam.random_coffee.events.model.{ EventId, RandomCoffeeEvent, UserId }
 import com.epam.random_coffee.events.repo.EventRepository
 import com.epam.random_coffee.events.services.EventService
 
+import java.time.Instant
+import java.time.temporal.ChronoUnit
 import java.util.UUID
 import scala.concurrent.{ ExecutionContext, Future }
 
 class EventServiceImpl(repo: EventRepository)(implicit ec: ExecutionContext) extends EventService {
 
-  override def create(eventName: String): Future[Event] = {
+  override def create(
+    name: String,
+    description: String,
+    eventDate: Instant,
+    author: String
+  ): Future[RandomCoffeeEvent] = {
     val id = EventId(UUID.randomUUID().toString)
-    val event = Event(id, eventName)
-    repo.save(event).map(_ => event)
+    val createdAt = Instant.now().truncatedTo(ChronoUnit.MINUTES)
```

Comment:
Not that important, but I think we should truncate to seconds

Reply:
Done